### PR TITLE
i18n(fr) Update and translated `aliases.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)
Update an trad last aliases file

### CODE REVIEW

**Comment by McFlyPartages** [comment]
LGTM

File: src/content/docs/fr/guides/aliases.mdx [hunk 1]
```diff
@@ -1,44 +1,46 @@
 ---
 title: Alias
 description: Une introduction aux alias avec Astro.
+i18nReady: true
 ---
 
 Un **alias** est une façon de créer des raccourcis pour vos imports.
 
 Les alias peuvent aider à améliorer l'expérience de développement dans les codebases avec de nombreux dossiers ou importations relatives.
```
Comment (pioupia) [inline src/content/docs/fr/guides/aliases.mdx:9]: I don't know if we shouldn't translate the `codebases` word...

File: src/content/docs/fr/guides/aliases.mdx [hunk 2]
```diff
@@ -1,44 +1,46 @@
...
 ---
-// Exemple: my-project/src/pages/about/company.astro
-
 import Button from '../../components/controls/Button.astro';
 import logoUrl from '../../assets/logo.png?url';
 ---
 ```
 
-Dans cet exemple, le développeur aurait besoin de connaître le chemin d'arborescence entre `src/pages/about/company.astro`, `src/components/controls/Button.astro`, et `src/assets/logo.png`. De plus, si le fichier `company.astro` venait à être déplacé, le chemin de chacun de ses imports devrait également être mis à jour.
+Dans cet exemple, un développeur devra comprendre la relation arborescente entre `src/pages/about/company.astro`, `src/components/controls/Button.astro`, et `src/assets/logo.png`. Et si le fichier `company.astro` devait être déplacé, ces importations devraient également être mises à jour.
```
Comment (pioupia) [inline src/content/docs/fr/guides/aliases.mdx:18]:
```suggestion
Dans cet exemple, un développeur devra comprendre la relation de l'arborescente entre `src/pages/about/company.astro`, `src/components/controls/Button.astro`, et `src/assets/logo.png`. Et si le fichier `company.astro` devait être déplacé, ces importations devraient également être mises à jour.
```

**Review by pioupia — APPROVED** [review]
2 comments, but otherwise the rest LGTM!

File: src/content/docs/fr/guides/aliases.mdx (hunk 1 above)
Comment (McFlyPartages) [reply src/content/docs/fr/guides/aliases.mdx:9]: I dont know

Comment (thomasbnt) [reply src/content/docs/fr/guides/aliases.mdx:9]: I think is good like this

**Pull request merged**